src/TorSharp/Tools/Tor/TorFetcher.cs

```diff
@@ -74,5 +100,59 @@ private FileNamePatternAndFormat GetFileNamePatternAndFormat()
 
             return new FileNamePatternAndFormat(pattern, format);
         }
+
+        public async Task<DownloadableFile> GetLatestDownloadableArmFileAsync(HttpClient httpClient,
+            Uri baseUrl,
+            FileNamePatternAndFormat patternAndFormat,
+            CancellationToken token)
+        {
+            SyndicationFeed syndicationFeed;
+            using (var response = await httpClient.GetAsync(baseUrl, HttpCompletionOption.ResponseContentRead, token).ConfigureAwait(false))
+            {
+                response.EnsureSuccessStatusCode();
+
+                using (var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
+                {
+                    var streamReader = new StreamReader(stream);
+                    var xmlReader = XmlReader.Create(streamReader);
```

Comment:
Nit: using around both `streamReader` and `xmlReader` (generally I `using` ALL disposables so future refactorings don't leak the true disposable resource -- the network stream)